### PR TITLE
Updates deprecated node installation method

### PR DESCRIPTION
**Pull request by lamoglia**
Following instructions from https://github.com/nodesource/distributions

### CODE REVIEW

**Comment by Remi-Gau** [comment]
We use neurodocker to generate the dockerfile: preferable to modify the makefile in this case

**Comment by Remi-Gau** [comment]
Will have a look at more details in what you suggest to check what can be added to it.

Note that I also opened an issue on neurodocker repo to facilitate installing the bids validator in any docker recipe, so I probably should try to reuse your suggestion in this other repo.

**Comment by Remi-Gau** [comment]
Note to myself:
- circle CI should run on PRs

File: Dockerfile [hunk 1]
```diff
@@ -3,13 +3,15 @@
 FROM ubuntu:jammy-20221130
 RUN apt-get update -qq \
            && apt-get install -y -q --no-install-recommends \
-                  ca-certificates curl apt-utils \
+                  ca-certificates curl apt-utils gnupg \
            && rm -rf /var/lib/apt/lists/*
-RUN curl -sL https://deb.nodesource.com/setup_18.x | bash -
+RUN mkdir -p /etc/apt/keyrings
+RUN curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | gpg --dearmor -o /etc/apt/keyrings/nodesource.gpg
+ENV NODE_MAJOR 18
```
Comment (Remi-Gau) [inline Dockerfile:10]:
```suggestion
ENV NODE_MAJOR 18 
```

Comment (Remi-Gau) [reply Dockerfile:10]: dummy change to see if circle ci runs

File: Dockerfile [hunk 2]
```diff
@@ -3,13 +3,15 @@
 FROM ubuntu:jammy-20221130
 RUN apt-get update -qq \
            && apt-get install -y -q --no-install-recommends \
-                  ca-certificates curl apt-utils \
+                  ca-certificates curl apt-utils gnupg \
            && rm -rf /var/lib/apt/lists/*
-RUN curl -sL https://deb.nodesource.com/setup_18.x | bash -
+RUN mkdir -p /etc/apt/keyrings
```
Comment (Remi-Gau) [inline Dockerfile:8]:
```suggestion
RUN mkdir -p /etc/apt/keyrings 
```

**Review by Remi-Gau — CHANGES_REQUESTED** [review]
yup the changes make sense to me

just better to transfer those changes into the makefile: https://github.com/bids-apps/base_validator/blob/master/Makefile

and then let neurodocker take care of writing the dockerfile and the extra boilerplate

let me know if you need pointers on how to do that

**Comment by lamoglia** [comment]
Ok, will do.

**Pull request closed without merging**